README.md

```diff
@@ -49,9 +49,9 @@ See the [Getting Started Guide](https://minikube.sigs.k8s.io/docs/start/)
 
 See https://minikube.sigs.k8s.io/docs/
 
-## More Examples
+## Minikube in action
 
-See our [examples page](https://minikube.sigs.k8s.io/docs/examples/)
+See minikube in action [here](https://minikube.sigs.k8s.io/docs/handbook/controls/)
```

Comment:
lets add an alias 
so anyone linked to this would be reffered to old location

Reply:
here is example of alias https://github.com/kubernetes/minikube/pull/7485

Reply:
do you mean people with the old link would be referred to the new page?

Reply:
@medyagh, I have added the alias and confirmed that it works using the netlify link.